lumicks/pylake/fitting/detail/model_implementation.py

```diff
@@ -732,6 +736,66 @@ def marko_sigga_ewlc_solve_distance_equation_tex(f, Lp, Lc, St, kT=4.11):
                                f"{dLc} - {FSt}", "d", lhs)
 
 
+def inverted_simplified_marko_sigga_coefficients(f, Lp, Lc, kT):
```

Comment:
The implementation functions have `sigga` as a typo.

Reply:
Whoops! Fixed!